config/environment.default.js

```diff
@@ -18,9 +18,17 @@ module.exports = {
   // Caching settings
   cache: {
     // NOTE: how long should objects be cached for by default
-    msToLive: 15 * 60 * 1000, // 15 minutes
+    msToLive: {
+      default: 15 * 60 * 1000, // 15 minutes
+      exportToZip: 5 * 1000 // 5 seconds
```

Comment:
It's unclear to me what these new settings in `environment.default.js` are for. Are these an unrelated change, or maybe local configuration?

Reply:
This PR includes the same response cache refactor @LotteHofstede's #349 PR is based on. It adds the ability to cache certain responses for different lengths of time. (e.g. you may want to cache most responses for 15 minutes, but for certain admin features you may want to disable the cache, or cache it a very short time) However the exportToZip config _is_ something that snuck in from an unrelated branch i.e. the export to zip feature that is on hold atm waiting for the [scripts and processes work](https://github.com/DSpace/Rest7Contract/pull/17) on the REST side. We'll remove it